CONTRIBUTING.md

```diff
@@ -88,10 +88,12 @@ We have yet to test the compatibility of the iaf-test module with Jetty. Until t
 - Close Welcome.
 - Make sure that the default text file line delimiter is set to Unix and default encoding is set to UTF-8: Window, Preferences, General, Workspace, New text file line delimiter: Unix, Text file encoding: UTF-8.
 - Make sure Maven is able to access the internet. E.g. when behind a proxy: Window, Preferences, Maven, User Settings, settings.xml should exist and contain proxy configuration.
-- Window, Open Perspective, Other..., Git, OK, Clone a Git repository, URI: https://github.com/ibissource/iaf.git, Next, Next, Finish.
-- Optionally (when you have access to the proprietary jars some modules depend on) add your Nexus credentials and enable the proprietary profile in your maven settings.xml
-- Right click iaf, Import projects..., Next, **deselect**: iaf-coolgen, iaf-ibm, iaf-ifsa, iaf-sap, iaf-tibco and iaf-idin (unless you have access to the proprietary repository), Finish.
-- Window, Open Perspective, Other..., Java EE.
+- Optionally (when you have access to the proprietary jars some modules depend on) add your Nexus credentials and enable the proprietary profile in your maven settings.xml.
+- Open a command prompt. Browse to the directory of your Eclipse workspace. Clone the IAF project from GitHub: `git clone https://github.com/ibissource/iaf` (you may need to install and configure Git before you can use it from the command-line). Within your Eclipse workspace directory, you should have created a subdirectory "iaf".
```

Comment:
Hier ga je er van uit dat iedereen weet wat Git is en hoe het werkt. Voorheen kon men door middel van wat knopjes in Eclipse ook het project uitcheckken. Waarom heb je er voor gekozen dit te vervangen?

Reply:
Laurens liep vast toen hij de instructies volgden zoals ze waren. Hij zag als sub-project namen niet de Maven artifact namen, maar iets anders. Ik vermoed dat Eclipse de projecten niet als Maven projecten herkende.

De doelgroep van CONTRIBUTING.md is Java ontwikkelaars, want we vertellen hier hoe je kan helpen bij de ontwikkeling van het framework. Voor die mensen zou het installeren en het gebruiken van Git geen probleem moeten zijn, denk ik. Zoals het er stond krijgen ontwikkelaars veel meer problemen.

Reply:
Qua doelgroep ben het met je eens, maar als je via Eclipse het als GIT project importeerd dan is alles mooi gesynchronizeerd en kan je zelfs vanuit Eclipse dingen committen. 

De beschrijving is toch heel duidelijk? Maar men moet hem wel lezen :)

- Window, Open Perspective, Other..., Git, OK, Clone a Git repository, URI: https://github.com/ibissource/iaf.git, Next, Next, Finish.

Reply:
Eens met Niels